src/include/catalog/database_catalog.h

```diff
@@ -246,7 +247,85 @@ class DatabaseCatalog {
   std::vector<std::pair<common::ManagedPointer<storage::index::Index>, const IndexSchema &>> GetIndexes(
       common::ManagedPointer<transaction::TransactionContext> txn, table_oid_t table);
 
+  /**
+   * Creates a language entry into the pg_language table
+   * @param txn transaction to use
+   * @param lanname name of language to insert
+   * @return oid of created entry if successful else INVALID_LANGUAGE_OID
+   */
+  language_oid_t CreateLanguage(const common::ManagedPointer<transaction::TransactionContext> txn,
+                                const std::string &lanname);
```

Comment:
Probably an insufficient API prototype (see comment in catalog accessor header).